test/gtest/ucs/test_datatype.cc

```diff
@@ -690,26 +709,6 @@ UCS_TEST_F(test_datatype, ptr_array_locked_random) {
     ucs_ptr_array_locked_cleanup(&pa);
 }
 
-UCS_TEST_F(test_datatype, ptr_array_locked_placeholder) {
```

Comment:
why this test was removed?

Reply:
@dmitrygx placeholder was an unused functionality, removed in #4860 to make room for the for-each optimization. This test is no longer relevant.